index.js

```diff
@@ -2,5 +2,9 @@ module.exports = {
   // Mainnet
   '1': '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
   // Ropsten
-  '3': '0x112234455c3a32fd11230c42e7bccd4a84e02010',
+  '3': '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
+  // Rinkeby
+  '4': '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
```

Comment:
I'm just now noticing the inconsistent case 🤔 Is this intentional?

Reply:
Yes, it's the checksum format